src/common-utils/devcontainer-feature.json

```diff
@@ -1,6 +1,6 @@
 {
     "id": "common-utils",
-    "version": "1.1.6",
+    "version": "1.2.6",
```

Comment:
```suggestion
    "version": "1.2.0",
```

Reply:
Sorry, it was late when I made this edit!